#using pythonBox exception handling is done